bin/end.fullchain

```diff
@@ -0,0 +1,113 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
demo certificate file, copied from tungstenite demo - should be removed again in the future. We'll want to use our self-signed certificate (see #364 )